app/views/teams/move_to_directorate.html.slim

```diff
@@ -11,3 +13,5 @@
 
 = render partial: 'browse_directorates'
 
+br
+= link_to "Back", team_path(@team), class: 'govuk-back-link'
```

Comment:
Do we need this here? Just thinking does that mean all processes have to have a back button at the bottom?

Reply:
@digitalali-moj ,  in this case, let me remove this bottom one ( I just thought it would be convenient 😳 ) to make this page in line with others